components/SlateRenderer/SlateRenderer.tsx

```diff
@@ -81,7 +49,39 @@ function SlateRenderer({ nodes }: Props) {
 
     return (
         <div className={styles.renderer}>
-            <Renderer nodes={nodes} components={components} />
+            <Renderer nodes={nodes}>
+                <Component match={isAttachmentNode} component={Attachment} />
+                <Component
+                    match={isContactNode}
+                    component={({ node: { contact } }) => (
+                        <ContactCard
```

Comment:
I think it is better to extract it to components folder like others

Reply:
Agree. I've ported your changes from #121